mongodb-setup/README.md

```diff
@@ -0,0 +1,38 @@
+# MongoDB support
+
+Connection and user login with a MongoDB database.
+
+This project uses the future [route v2 convention](https://remix.run/docs/en/main/pages/v2#file-system-route-convention).
```

Comment:
Please revert to v1 routing convention, so it's in line with the rest of the examples